platform/src/test/resources/test-anchor-config.yaml

```diff
@@ -27,7 +27,6 @@ app-config:
     jwtSecretKey: secret
 
   # sep-1
-  sep1:
```

Comment:
❗  Wait, I think you removed this by mistake?

Reply:
Fixed.